services/data_plane_manager/src/main/resources/machine.json

```diff
@@ -0,0 +1,64 @@
+{
+  "Hosts": [
```

Comment:
Remove the directory services/data_plane_manager/src/main/resources, unless you really need those files.